src/sentry/models/grouptombstone.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+import logging
+
+from django.db import models
+
+from sentry.constants import LOG_LEVELS, MAX_CULPRIT_LENGTH
+from sentry.db.models import BoundedPositiveIntegerField, FlexibleForeignKey, Model
+
+
+class GroupTombstone(Model):
+    __core__ = False
+
+    previous_group_id = BoundedPositiveIntegerField(unique=True)
+    project = FlexibleForeignKey('sentry.Project')
```

Comment:
Oops, you're right — for some reason I was thinking this was an integer field and apparently wrote this down without reading. Sorry!